R/palaeorotate.R

```diff
@@ -92,6 +96,11 @@
 #'   - 0--540 Ma (grid)
 #'   - 0--750 Ma (point)
 #'
+#' While access is provided to all Global Plate Models available via the
+#' GPlates API, the use of the following mantle reference frame models is not
+#' recommended for reconstructing coordinates:
+#' "MULLER2022", "MULLER2019", and "MATTHEWS2016_mantle_ref".
+#'
 #' @section References:
```

Comment:
Should these references be in the same order as the models?

Reply:
This section has been updated. Let me know if you still have concerns!

Reply:
I think keeping them in alphabetical order is fine (I guess this is just a style preference).